python/tvm/relay/op/contrib/forge/forge_passes.py

```diff
@@ -1891,7 +1891,10 @@ def callback(self, pre, post, node_map):
         act = node_map[self.act][0]
         axis = post.attrs.axis
         input_shape = [int(dim) for dim in pre.args[0].checked_type.shape]
-        adjusted_axes = [(ax - len(input_shape)) if ax >= 0 else ax for ax in axis]
+        if axis is None:
```

Comment:
Is this expected behavior? How does PyTorch interpret this? 

Please document this behaviour as well

Reply:
Axis is None cases occurs in squeeze_() cases in an in-place operation that removes all dimensions of size 1 from a tensor in pytorch.